### ISSUE TITLE
Angular module routing icon for any module, not just main

### ISSUE THREAD

**Issue by jagomf** [vscode-icons/vscode-icons#2702]
- [x] I'm sure this issue is not a _duplicate_.

Currently main Angular routing file `app-routing.module.ts` does have a differentiated icon (`'ng_routing_ts'`).

However, in Angular any module can have a routing file and be routed to. However the existing icon for Angular routing is not currently being assigned, thus rendering the currently visible icon without any difference to the module file.

Here you can notice the Users module routing file icon is the same as the Users module file icon itself, despite the main App module routing file icon actually being differentiated:
![image](https://user-images.githubusercontent.com/4032522/108996518-c5c71100-769e-11eb-9aca-1f44ff1e7927.png)

So my proposal is **to enable any module routing file in an Angular project to display the icon being currently used for main App module routing**.

- Language Extensions affected: `-routing.module.ts`, `-routing.module.js`, `-routing.module.dart`

**Comment by robertohuertasm** [comment]
Unfortunately, this doesn't work like that. We still haven't got support for wildcards in filenames or extensions and this is due to a well-known vscode api limitation, so it's not on us.

As an alternative, remember you can always use the customization functionalities and associate the icon you want to the filename of your choice. That, though, should be a manual process on your side, probably per project. No other workaround for the moment. 😞 Sorry.

**Comment by robertohuertasm** [comment]
Reopening here just to discuss about any alternatives. What about supporting `.routing.module.*`? We're already supporting it `routing.*`... Not sure though...

**Comment by jagomf** [comment]
> Reopening here just to discuss about alternatives. What about supporting `.routing.module.*`? We're already supporting it `routing.*`... Not sure though...

Actually when you create a new module `'foo'`, the default naming for the routing file of that module will already be `'foo-routing.module.ts'`, so I guess `'.routing.module.*'` would not make it. We would rather need `'*routing.module.ts'`, or more explicitly `'*-routing.module.ts'`.

**Comment by KingDarBoja** [comment]
Yeah, Angular schematics will always generate the routing module with that format, so the lack of wildcards from vscode api makes it impossible to support these filenames 😢
